backends/graphics/opengl/texture.cpp

```diff
@@ -530,6 +530,46 @@ void TextureRGBA8888Swap::updateGLTexture() {
 	// Do generic handling of updating the texture.
 	Texture::updateGLTexture();
 }
+
+TextureARGB8888::TextureARGB8888()
+    : FakeTexture(GL_RGBA, GL_RGBA, GL_UNSIGNED_BYTE, Graphics::PixelFormat(4, 8, 8, 8, 8, 0, 8, 16, 24)) // ABGR8888
```

Comment:
I am worrying that some big endian GLES systems might not support this format, which may be why we have a fake texture for it. So maybe when `SCUMM_BIG_ENDIAN` is defined (or when `SCUMM_LITTLE_ENDIAN` is not defined) you should be converting to RGBA8888 rather than ABGR8888.

Reply:
I actually forgot that you only added the format in `getSupportedFormats()` when SCUMM_LITTLE_ENDIAN is defined. This means we probably cannot play AGS games with OpenGL/GLES currently on big endian systems, but at least that means that `getGLPixelFormat()` and the `TextureARGB8888` are consistent with it.

It would be nice to extend the changes to the big endian case, but if you cannot test that case and are not confortable with doing it, you could just mention in the commit message that this is done only for little endian systems.